website/content/docs/connect/transparent-proxy.mdx

```diff
@@ -38,9 +38,13 @@ Otherwise, all Connect services can talk to all other services.
 The Kubernetes integration takes care of registering Kubernetes services with Consul, injecting a sidecar proxy, and
 enabling traffic redirection.
 
+## Upgrading to Transparent Proxy
+
+When upgrading from older versions (i.e Consul-k8s < `0.26.0` Consul-helm < `0.32.0`) to Consul-k8s >= `0.26.0` and Consul-helm >= `0.32.0`, please make sure to follow the upgrade steps [here](/docs/upgrading/upgrade-specific/#transparent-proxy-on-kubernetes).
```

Comment:
Wondering if we can make this a `~>` for the line so that it's really boldly called out?

Reply:
Good idea!